app/routes/profile/$email/index.tsx

```diff
@@ -9,16 +9,38 @@ import {
   Typography,
   Alert,
   AlertTitle,
+  TextField,
+  Button,
 } from "@mui/material";
-import Card from '@mui/material/Card';
-import CardContent from '@mui/material/CardContent';
-import { getGitHubProfileByEmail, getGitHubProjectsByEmail } from "../../../models/profile.server";
-import {  useLoaderData } from "@remix-run/react";
-import type { LoaderArgs, LoaderFunction } from "@remix-run/node";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import {
+  getGitHubProfileByEmail,
+  getGitHubProjectsByEmail,
+} from "../../../models/profile.server";
+import { useLoaderData } from "@remix-run/react";
+import { ActionFunction,
+  type LoaderArgs,
```

Comment:
here you need to define  LoaderArgs, LoaderFunction as types like this 
`import type { LoaderArgs, LoaderFunction } from "@remix-run/node";`